src/Global.h.Rt

```diff
@@ -65,10 +65,10 @@ for (m in Margin) { ?>
     for (m in NonEmptyMargin) { ?>
```

Comment:
I think `FTabs` should be moved from here. Probably to `CartLatticeContainer.h`

Reply:
Closed in [f47e01e](https://github.com/CFD-GO/TCLB/pull/454/commits/f47e01eca95590835f8efeb77e917ea56261d979)

Where should we put the definitions for FTabs though? Just inline them?

Reply:
@llaniewski verbally confirmed they can stay in `Lattice.cu`